pkg/linter/lintRedundantTables.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2023- IBM Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package linter
+
+import (
+	"fmt"
+
+	"github.com/np-guard/vpc-network-config-analyzer/pkg/vpcmodel"
+)
+
+const redundantTablesName = "redundant tables"
+const securityGroup = "security group"
+const networkACL = "network ACL"
```

Comment:
these two last constants are already declared in another package.
expose and reuse.
also fix FilterKindName() to use these constants

Reply:
1. Right. Using `FilterKindName()` no need for the constants here
2. `FilterKindName()` can not use these constants since they are defined in `ibmvpc/commonvpc`
3. Perhaps we should modify the code in `ibmvpc/commonvpc` to use `FilterKindName()` - but not in this PR @adisos do you want to open a followup issue to handle this?

Reply:
yes, please open a separate issue for this.

Reply:
#724